packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/audit/ITAuditTest.java

```diff
@@ -1248,4 +1249,34 @@ public void testEncoding() throws PangeaAPIException, PangeaException {
 		final var response = clientGeneral.log(event, null);
 		assertTrue(response.isOk());
 	}
+
+	@Test
+	void testOpenApiSchemaValidation() {
```

Comment:
Should we add tests were schema validation success?

Reply:
Added.